src/components/staking/PageValidator.vue

```diff
@@ -14,9 +14,11 @@
         color="secondary"
         @click="$router.push(`/validators`)"
       >
-        <img src="/img/icons/arrow-icon.svg" />
-        <div style="width: 20px;display: inline-block"></div>
-        Back to Validators
+        <div style="display:flex; flex-direction:row; align-items: center;">
+          <i class="material-icons arrow">arrow_back</i>
+          <div style="width: 20px;display: inline-block;"></div>
```

Comment:
just want to flag this - we shouldn't use empty divs to achieve stylistic outcomes. this should just be padding on the right of the arrow. 

```
.arrow {
  padding-right: 20px;
}
```

Reply:
Yes, you are right. I am currently working on my CSS skills, I promise xD

Reply:
https://github.com/luniehq/lunie/pull/3398